motioneye/mediafiles.py

```diff
@@ -1029,18 +1030,18 @@ def get_current_picture(camera_config, width, height):
 
     image.thumbnail((width, height), Image.CUBIC)
 
-    sio = io.StringIO()
-    image.save(sio, format='JPEG')
+    bio = BytesIO()
+    image.save(bio, format='JPEG')
 
-    return sio.getvalue()
+    return bio.getvalue()
 
 
 def get_prepared_cache(key):
     return _prepared_files.pop(key, None)
 
 
 def set_prepared_cache(data):
-    key = hashlib.sha1(str(time.time()).encode()).hexdigest()
+    key = sha1(str(time()).encode()).hexdigest()
```

Comment:
```suggestion
    key = sha1(str(time()).encode(), usedforsecurity=False).hexdigest()
```
This should placate codefactor.io which is using bandit under the hood.

Reply:
i'm a newbi does this "change/fix" will be reflected to the repository and coker image?

Reply:
Not sure whether it's good to tell the method that it's not for security, because it is. My idea instead was to mute the bandit annotation via comment for now:
```suggestion
    key = sha1(str(time()).encode()).hexdigest() # nosec B303
```